build.go

```diff
@@ -33,6 +34,8 @@ import (
 	"github.com/buildpacks/pack/internal/style"
 )
 
+const lifecycleImageRepo = "natalieparellano/lifecycle-image"
```

Comment:
Need to change this to be the correct one. I put it this way for now so that GHA can run.